### ISSUE TITLE
Xenophon Ten Thousand - Week 6

### ISSUE THREAD

**Issue by CianColgan** [ChiaraPalladino/furesearch#43]
**Progress Report:**
I feel motivated and like I made good progress this week, even though it's been extremely tedious and slow. Since our last meeting, I mapped the rest of Book 3 and about halfway through Book 4, past the hair-splittingly painful Carduchian Mountains and into Armenia, and I feel confident in the route and reasoning thus far. Additionally, I've been pouring through journal articles and datasets for context of confusing area, both those you sent me and what I've found on my own.

I'm in another confusing area right now, and I'll detail it in a comment below. I think I know how I'm going to proceed forward, but any input is much appreciated. It interacts with the core project principles of plotting the route Xenophon says vs. what they may have actually taken in a strange way, so I wanted to bring it to you attention.

**Comment by CianColgan** [comment]
# Xenophon is Lost: Armenian Rivers Edition

For context, this is the region we're dealing with:
![Image](https://github.com/ChiaraPalladino/furesearch/assets/170037960/4c8dd3d1-be06-4e45-a472-b16daf50ddc4)

## The Ideal (and most logical) Route:
![Image](https://github.com/ChiaraPalladino/furesearch/assets/170037960/0d043540-7a5d-4ed6-8a28-ddd4f76869a1)
This is the most logical route to finish out Book 4. I've mapped up to 4.5.2 where he crosses the headwaters of the Euphrates (which in the modern day isn't even considered the Euphrates, it's the Murat River, but Xenophon seems to [perhaps mistakenly] identify it as the Euphrates. This is but a taste of things to come). Ideally, he should pass through Gymnias, reach the Theches mountain, see the sea, and from there make his way to Trapesuz.

However, this is not what his measurements and landmarks have in mind.

## The Phasis Problem:
What first tipped me off to the impending confusion is that Xenophon mentions they encounter the Phasis river (4.6.4) well before they reach Gymnias (4.7.19). This is geographically impossible.

The general consensus is that the Phasis river is most commonly identified (including by Pleiades) as the modern day Rioni, which originates in the North Caucuses and flows into the Black Sea, way further north than anywhere they visit. If Xenophon did encounter this river, it would look like this:
![Image](https://github.com/ChiaraPalladino/furesearch/assets/170037960/ba85e838-fb68-46e4-9135-e95ddda9590f)

### So, evidently, it seems like Xenophon may have misidentified the Phasis, and he is also likely lost. 
While presumably Xenophon's pride doesn't allow him to explicitly admit to being lost, this section of the journey (4.5.2-4.7.19) is all about the Ten Thousand freezing and starving to death as they go from unnamed village to unnamed village, trying to find guides that won't mislead them, implying they are rather thoroughly lost. He also drops some truly bonkers measurements later on (5 stages/30 parasangs; 4.7.1; 7 stages/50 parasangs, 4.7.15), so I truly don't think he has any idea what's going on.

One of the maps I found [here](https://topwar.ru/uploads/posts/2019-02/1550991265_anabasis-big.jpg), while admittedly all in German, does show multiple different reconstructed routes in this area, so I'm not the first to notice this problem. 

Thankfully, in looking around, I found [this source](https://shs.hal.science/halshs-01523235/document#:~:text=Xenophon%20and%20Procopius%2C%20the%20authors,Sea%20by%20the%20Phasis%2D%20Rioni.) (Anca Dan. The Rivers Called ‘Phasis’. Ancient West & East , 2016, Volume dedicated to Professor Alexandru Avram to celebrate his 60th birthday, 15, pp.245-277. ff10.2143/AWE.15.0.3167476ff.
ffhalshs-01523235f) that sheds light on all the ancient geography surrounding the river Phasis, including that what ancient sources (Xenophon's _Anabasis_ among them) have called the Phasis have also been suggested to be the Phasis-Araxes [Aras], the Lykos [Kelkit Çayı], and the Boas [Çoruh Nehri]. 

The Lykos/Kelkit is way west of where Xenophon is, so I've eliminated it from the running. The other two are here:
![Image](https://github.com/ChiaraPalladino/furesearch/assets/170037960/95724c6b-1940-40fa-a7ae-b772fcf5588a)

Both of these seem reasonable for Xenophon to have encountered, much more reasonable than the Phasis-Rioni. 

## Solution 1 - Aras (My pick)
The most logical solution for me is that Xenophon identified the Phasis as the Aras. Xenophon would encounter it almost precisely when he says he encounters the Phasis in the text, the plenty of space northward for him to get hopelessly lost and drop those stupid high measurements, then eventually find his way to Gymnias and proceed as normal.

# The biggest question is whether or not he goes north, towards Gymnias, from where I have him plotted currently, or northwest. Right now, I'm leaning northwest.

## Solution 2 - Çoruh
Another solution is the Çoruh, but this doesn't seem possible. He would have to overshoot Gymnias to reach it, and from where I have him he'd also have to cross the Aras anyway, which he doesn't make mention of. He could get lost in the steep mountains that surround the river, but they don't seem to be conducive to his measurements nor terrain description.

Also, I have the Çoruh as the Harpasus he's said to encounter later, and while it is possible he could hit the same river twice and think it's two different ones, it seems more unlikely than just misidentifying the Aras.

**Comment by ChiaraPalladino** [comment]
Okay, I'm going to tackle this one first. 

* The problem of the Phasis river. You actually found the source I would have recommended, so all good there - and yes, the location of the Phasis is problematic. I agree that the Aras would seem the most logical solution, also given the other locations pinpointed in this area, **however: it could be worth it to calculate his route in both cases and give two alternative visualizations here, to give the reader the impression that this is a controversial passage. We may have to do it a few times in Book 3-4.** We could discuss later how to visualize these alternatives in a more divulgative version of the storymap. 
* What alternative locations do we have for Gymnias? 
* Finally, there are a couple of articles worth checking on this leg of the itinerary, if you haven't found them yet: **Brennan-Tuplin, Theches: an elusive mountain** (you can download the pdf using the Furman library credentials); and **Paradeisopoulos, A chronology Model for Xenophon's Anabasis** (here: https://www.researchgate.net/publication/289918350_A_Chronology_Model_for_Xenophon%27s_Anabasis)

**Comment by CianColgan** [comment]
Posting a quick update after spending all of today looking at this contextual research, finding a reading some additional sources, and comparing them to my map and my current choices. 
I'm surprised at how recent some of this work is, and I'm wondering how I wasn't able to find it sooner. Also, I stuck with just the same few authors that keep referencing each other, since their work seems trustworthy and is also most relevant to my current work.

**Sources:**
* [A Chronology Model for Xenophon's Anabasis](https://www.researchgate.net/publication/289918350_A_Chronology_Model_for_Xenophon's_Anabasis) - Paradeisopoulos, 2013
* [The Rivers Called ‘Phasis'](https://shs.hal.science/halshs-01523235/document#:~:text=Xenophon%20and%20Procopius%2C%20the%20authors,Sea%20by%20the%20Phasis%2D%20Rioni) - Dan, 2022
* [From the Tigris to the Sea](https://www.academia.edu/113341480/From_the_Tigris_to_the_Sea_The_Problematic_Geography_of_Anabasis_Book_4?rhid=29106370978&swp=rr-rw-wc-110042642) - Tulpin, 2022
* [Theches: an elusive mountain](https://www.researchgate.net/publication/371394138_Theches_an_elusive_mountain) - Brennan and Tulpin, 2023
* [The Retreat of the Ten Thousand from River Harpasos to Gymnias, Mount Theches and Trapezus](https://www.academia.edu/110042642/The_Retreat_of_the_Ten_Thousand_from_River_Harpasos_to_Gymnias_Mount_Theches_and_Trapezus) - Paradeisopoulos, 2023
(Also I've been trying to access [Mind the Gap: A ‘Snow Lacuna’ in Xenophon’s Anabasis?](https://brill.com/display/book/edcoll/9789004234192/B9789004234192-s011.xml), but none of the institution logins have worked and it also may not be directly relevant for plotting a route since it deals with a timeline.

### Gymnias
* As for the location of Gymnias, there's only two major ones I've seen:
    * The seemingly most commonly agreed upon location is modern Bayburt. This is slightly northwest of the Pleiades reference point, but it's what I and most others are operating off of.
    * Another location (most prominently argued by Paradeisopoulos) is Gymuri, which is way further east

### Paradeisopoulos 
![download (1)](https://github.com/ChiaraPalladino/furesearch/assets/170037960/99afe770-b757-4048-9944-6a9f12333dc0)
* Paradeisopoulos specifically argues for an unorthodox but plausible route that takes Xenophon much further east before circling back
    * It seems very convincing given how much it makes sense of some of Xenophon's wilder measurements. However, it's also strongly predicated on Diodorus' account of the Ten Thousand's retreat a few centuries later, which differs only slightly but specifically in this section
* He also has this map of caravan route/roads in the area, and I would absolutely _love_ to know where he got this data because it would make my life so much easier since I've been building a map with no knowledge of contemporary local roads whatsoever
* Paradeisopoulos also identifies almost every river differently than how I have with my research, though these river identifications don't seem to be major points of contention with other scholars so maybe I'm just the weird one??
    * He seems to think that Xenophon's Tigris headwaters are the Mansur Creek, both the Teleboas and the Euphrates headwaters are the East Euphrates/Murat, and he argues the Harpasos is Arpa [which he also seems to misidentify as the Akhuryan, a completely separate river, but also brings up the Arpa later as if this isn't contradictory??]
    * Meanwhile I identify Xenophon's Tigris headwaters as Yanarsu Stream, the Teleboas as the Karasu [of which I'm now seeing there might be two different rivers both named Karasu. I hate this], the Euphrates headwaters as the Murat, and the Harpasos as the Çoruh
    * The only one we (and everyone else) seem to agree upon is that Xenophon misidentified the Phasis as the Aras/Araxes
* Paradeisopoulos takes a slightly different route through Western Armenia than I do, on account of these river identifiers, but we both end up in roughly the same area anyhow

In conclusion I feel some sort of gut-instinct disagreement with many of Paradeisopoulos' identifiers, but I do think his conclusion has merit and does help explain a lot of Xenophon's inconsistencies. Not helping this is that he actually includes useful maps for Xenophon's route in a way most similar to how I'm trying to build, which is extremely helpful, something the other scholars do not.
